docker/base/centos7/Dockerfile

```diff
@@ -82,7 +82,8 @@ RUN yum -y update \
 RUN wget https://archive.apache.org/dist/ant/binaries/apache-ant-1.9.15-bin.tar.gz \
     && tar xvf apache-ant-1.9.15-bin.tar.gz -C /opt \
     && ln -s /opt/apache-ant-1.9.15 /opt/ant \
-    && sudo ln -s /opt/ant/bin/ant /usr/bin/ant
+    && sudo ln -s /opt/ant/bin/ant /usr/bin/ant \
```

Comment:
Super minor nit. apache-any -> apache-ant.

Reply:
Hmm. Maybe this comment is for an old commit?  I don't see `apache-any` in this file currently.  I think I might be missing something, sorry.

Reply:
Sorry I wasn't clearly enough. The commit message itself has that typo:
```
@SteVwonder
base: ensure apache-any tarball is cleaned up
```